Health/Presentation/Personal/Views/MainView/RecommendPlaceCell.swift

```diff
@@ -64,7 +64,19 @@ class RecommendPlaceCell: CoreCollectionViewCell {
         locationLabel.text = course.sigun
         distanceLabel.text = "\(course.crsDstnc)km"
         durationLabel.text = course.crsTotlRqrmHour.toFormattedDuration()
-        levelLabel.text = getLevelText(from: course.crsLevel)
+        // 난이도 텍스트와 이미지 색상 동시 설정
+        let levelInfo = getLevelText(from: course.crsLevel)
```

Comment:
이젠 `getLevelInfo`가 맞을 것 같네요
```suggestion
        // 난이도 텍스트와 이미지 색상 동시 설정
        let levelInfo = getLevelInfo(from: course.crsLevel)
```